app/web/src/components/EditorMenu/EditorMenu.tsx

```diff
@@ -0,0 +1,126 @@
+import { Menu } from '@headlessui/react'
+
+import { useIdeContext, ideTypeNameMap } from 'src/helpers/hooks/useIdeContext'
+import Svg from 'src/components/Svg/Svg'
+import { useRender } from 'src/components/IdeWrapper/useRender'
+import { makeStlDownloadHandler, PullTitleFromFirstLine } from './helpers'
+
+const EditorMenu = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/29681384/121826268-436c6680-ccfa-11eb-9b06-ba4d61270ab5.png)

Reply:
:)
Yeah I did.
CContext is super important when trying to look at other's code. You could have figured this out yourself but 20x quicker for me to add a few screen shots.